### PR TITLE
Adds Flippo's to the ShadySmokes

### PR DESCRIPTION
**Pull request by Eden077**
## About the PR

Number of basic lighters has been increased to three, Flippo has been added in every shadysmokes. This adds up to 3 basic lighters and 2 Flippo's (Engraved Flippo was removed due to it being a thief objective)

## Why / Balance

I don't see any issues with letting people have more ease of access with wanting to smoke cigerettes, It's not like it's going to crash the server (Unless it fucking does)

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Two Flippo lighters have been added to ShadySmokes along with three basic lighters!

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/cigs.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@
     Matchbox: 5
     PackPaperRollingFilters: 3
     CheapLighter: 4
-    Lighter: 1
+    Lighter: 3
+    FlippoLighter: 2
+    FlippoEngravedLighter: 1
```
[the remark this answers is not in the document]
Comment (Eden077) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/cigs.yml:17]: Of course it is... WHY WOULDN'T IT BE!!!

[the remark this answers is not in the document]
Comment (Eden077) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/cigs.yml:17]: Let me fix that

**Comment by Eden077** [comment]
The silly little engraved Flippo error has been resolved

**Comment by OctoRocket** [comment]
Objection: map it instead so that it's more unique

**Comment by Eden077** [comment]
> Objection: map it instead so that it's more unique

Engraved Flippo has been mapped, it's in detectives locker

**Comment by OctoRocket** [comment]
I meant map the flippo over putting three of them in every smoke vender because then it becomes a unique find. Maybe put them on salv wrecks too

**Comment by Eden077** [comment]
Nah, I'd rather just let people actually have a 100% chance to use them

**Comment by OctoRocket** [comment]
The main reason flippos exist are to look fancier and make you look rich

**Comment by Eden077** [comment]
So just play detective lol, not arguing over cool looking lighters that people wanna use

**Comment by OctoRocket** [comment]
Zippos are often collector's items irl and having them be abundant everywhere on the station doesn't reflect that

**Comment by Eden077** [comment]
As I said, I'm not arguing this with you.

**Comment by OctoRocket** [comment]
> So just play detective lol, not arguing over cool looking lighters that people wanna use

It's entire purpose is to be an exclusive item used by the wealthier members of the station (command, warden, etc), that's how it is in 13

**Comment by OctoRocket** [comment]
> As I said, I'm not arguing this with you.

So you're not going to defend the merging of this PR?

**Comment by Eden077** [comment]
This is not space station 13, please stop having an aggressive outlook on **MY** PR and let the mantainers merge if they want it

**Comment by OctoRocket** [comment]
I'm trying to give criticism as to the reasons why this doesn't make much sense. At a very base level the flippo has 4 more welding fuel than the basic lighter so there shouldn't be the same number of them in the vender as there are basic lighters.

**Comment by Eden077** [comment]
Balancing concerns aren't exactly viable on a lighter, it's ease of access for characters who smoke a lot in an RP enviroment, this is HRP please take a balancing concern such as that elsewhere.

**Comment by OctoRocket** [comment]
It still doesn't make much sense that not only is there a lighter that is a direct upgrade to the basic lighter that comes at the same frequency as the basic lighter, but that the cheap lighter has the same amount of stock as both of it's direct upgrades. I feel like the balance where more people have a cheap lighter works well, as it can lead to people asking other people for a light RP wise. The basic lighter should be a be a nice upgrade for people who smoke more often, and that the zippo should be more rare for people who smoke a lot. Maybe a character who looks in maints finds it and trades it off to an engineer for some building materials or the such. Having a nicer and rarer lighter has a lot of RP potential if someone who doesn't smoke finds it and wants to sell or trade it off.

**Comment by Eden077** [comment]
It is... a fun PR... for lighters that are not currently obtainable... please stop arguing with me.

**Comment by CrigCrag** [comment]
i think maints loot would be a better spot for these

**Comment by Eden077** [comment]
I mean I'm adding them so that characters who would rather use Flippo's have easy access to them, it's purely ease of access and adding them into the actual game

**Comment by Nairodian** [comment]
My only comment is to adjust it to 2. I'm just fine with it being in a vendor but to make it flow nicely still, 5 match boxes, 4 basic lighters, 3 high fuel lighters, 2 flippo, (they actually hold more than a high fuel lighter!).

**Comment by Eden077** [comment]
Yeah I can... Do that... holy fuck where is my dev enviroment, give me a minute

**Comment by Eden077** [comment]
Alright, number has been changed

**Comment by OctoRocket** [comment]
> It is... a fun PR... for lighters that are not currently not obtainable... please stop arguing with me.

I really want to know why what I'm saying doesn't make sense (specifically the comment above this one).

**Comment by Eden077** [comment]
As I have already said _**multiple**_ times, please stop attempting to argue with me.

**Comment by OctoRocket** [comment]
I am going to be completely honest. I keep trying to bring up what I consider _reasonable_ and _well meaning_ critiques as well as bringing up alternative ways to add the flippo to the game, and all I feel like I'm getting in response is a lot of "I don't care at all about what you are saying, please shut up." Why?

**Comment by Eden077** [comment]
All checks have passed, hell yeah (Fuck you Ubuntu, not failing me this time!)

**Comment by LankLTE** [comment]
> I am going to be completely honest. I keep trying to bring up what I consider _reasonable_ and _well meaning_ critiques as well as bringing up alternative ways to add the flippo to the game, and all I feel like I'm getting in response is a lot of "I don't care at all about what you are saying, please shut up." Why?

I kindly ask that you stop trying to incite arguments. Eden has attempted to end the discussion several times, and clearly has no interest in discussing it further. Nobody is required to have discussion with you if they don't want, so in the future drop the topic when asked.

**Comment by xYAMIEx** [comment]
After reading through all of this, I agree with Nair. It would be lovely to have them in the vending machines, but not have _too_ many in them. Having just two per machine is a nice balance to make them more of a higher end lighter, but not have it so rare where people have to fight over them. The only issue that _may_ become a problem is cargo ordering a surplus of cigrestocks even if they are not needed, but that’s just a hypothetical issue that we cannot measure yet. If this does become a problem in the future, adding them to maint loot pools as a rarer piece of loot alongside the machine with **NO** way to restock them from the restock boxes could be a nice balance. 
sorry if this doesn’t make too much sense, I’m tired.

**Comment by Eden077** [comment]
> After reading through all of this, I agree with Nair. It would be lovely to have them in the vending machines, but not have _too_ many in them. Having just two per machine is a nice balance to make them more of a higher end lighter, but not have it so rare where people have to fight over them. The only issue that _may_ become a problem is cargo ordering a surplus of cigrestocks even if they are not needed, but that’s just a hypothetical issue that we cannot measure yet. If this does become a problem in the future, adding them to maint loot pools as a rarer piece of loot alongside the machine with **NO** way to restock them from the restock boxes could be a nice balance. sorry if this doesn’t make too much sense, I’m tired.

I think generally people aren't gonna waste money for funny cigs, there are so many ShadySmokes across station and also the amount of Flippo's has already been decreased so about half of this comment has already been resolved.

**Comment by Eden077** [comment]
Also as a small sidenote I never did add Flippo to the restock (If that's how restocks work, please correct me if I'm wrong) so there shouldn't be any issues with restocking the hell out of them

**Comment by xYAMIEx** [comment]
> > After reading through all of this, I agree with Nair. It would be lovely to have them in the vending machines, but not have _too_ many in them. Having just two per machine is a nice balance to make them more of a higher end lighter, but not have it so rare where people have to fight over them. The only issue that _may_ become a problem is cargo ordering a surplus of cigrestocks even if they are not needed, but that’s just a hypothetical issue that we cannot measure yet. If this does become a problem in the future, adding them to maint loot pools as a rarer piece of loot alongside the machine with **NO** way to restock them from the restock boxes could be a nice balance. sorry if this doesn’t make too much sense, I’m tired.
> 
> I think generally people aren't gonna waste money for funny lighters, there are so many ShadySmokes across station and also the amount of Flippo's has already been decreased so about half of this comment has already been resolved.

Yippie!!!

**Pull request merged**